src/bash.d/01-crew

```diff
@@ -3,6 +3,12 @@
 
 function _crew () {
   local CREW_PACKAGES_PATH="${CREW_PREFIX}/lib/crew/packages/"
+  local CREW_LOCAL_REPO_ROOT="$(git rev-parse --show-toplevel 2> /dev/null)"
+  if [[ -z "$CREW_LOCAL_REPO_ROOT" ]]; then
+    local CREW_LOCAL_REPO_ROOT_PACKAGES_PATH="${CREW_PACKAGES_PATH}"
```

Comment:
I'm not sure this is a good idea.  We want the crew repo and the local working repo for contributors separate, correct?  I suppose it could work but you would have to work with separate git remotes and that can get a bit dangerous if you accidentally push to the wrong repo.

Reply:
This only populates the autocomplete entries if you are in `CREW_LOCAL_REPO_ROOT`. So for instance if you have `ecm.rb` only in `CREW_LOCAL_REPO_ROOT/packages/` then autocomplete will show it, but only if your working directory is `CREW_LOCAL_REPO_ROOT`.

Note that `crew install ecm` or `crew build ecm` won't work even if you are in `CREW_LOCAL_REPO_ROOT`, because there is no `ecm.rb` in `/usr/local/lib/crew/packages`. (I think you were talking about getting crew to prompt in those cases about prompting to add... but that would have to be something inside `crew`.

Reply:
This is what happens when I touch `packages/ecm.rb` in a git clone dir. `crew install` autocomplete shows `ecm` but the install fails.
![image](https://github.com/chromebrew/crew-profile-base/assets/1096701/f483c24d-6dbe-419f-9333-17046d61a24f)

Reply:
OK, sorry, I wasn't fully understanding what was going on here.